src/test/java/uk/nhs/ctp/service/UserManagementServiceTest.java

```diff
@@ -75,6 +75,8 @@ public void createNewSupplierUser() {
     ReflectionTestUtils.setField(userManagementService, "cdss2", "http://cdss2.com");
     ReflectionTestUtils.setField(userManagementService, "dos", "http://dos.com");
     ReflectionTestUtils.setField(userManagementService, "logs", "http://elastic.com");
+    ReflectionTestUtils.setField(userManagementService, "fhirServer", "http://fhir-place.com");
+    ReflectionTestUtils.setField(userManagementService, "blobServer", "http://blob-palace.com");
```

Comment:
Blob Palace? 🏰

Reply:
it was a typo but I loved it so much I left it in